todo_app/__init__.py

```diff
@@ -0,0 +1,18 @@
+# Set env variables here 
+import requests
+import os
+from dotenv import load_dotenv
+
+host = 'https://api.trello.com'
```

Comment:
Given that you hardcode this value here - and there's no reason we expect users might want to configure this - we should skip setting it as an environment variable and just specify it directly in the app.py file too (or if we do keep this logic, we could pull it in as a constant)